src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -54,26 +48,19 @@ macro_rules! build_inconsistent_fs {
 const MY_TTL_SEC: u64 = 3600; // TODO: should be a long value, say 1 hour
 /// The generation ID of FUSE attributes
 const MY_GENERATION: u64 = 1; // TODO: find a proper way to set generation
-#[allow(dead_code)]
 /// The limit of transaction commit retrying times.
 const TXN_RETRY_LIMIT: u32 = 10;
 
 /// File system in-memory meta-data
 #[derive(Debug)]
 #[allow(dead_code)]
```

Comment:
Is this mark still available?

Reply:
Yes, the field named `cur_fd` is never used. I don't know what is it.